File: 2022/AllFlowerAdmin/app/src/main/java/com/admin/allflower/util/ProgressUtil.java
package com.admin.allflower.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.admin.allflower.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2026/Ddpick/app/src/main/java/com/creator/ddpick/util/ProgressUtil.kt
package com.creator.ddpick.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.ddpick.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}